source/extensions/filters/http/cache/cache_headers_utils.cc

```diff
@@ -0,0 +1,157 @@
+#include "extensions/filters/http/cache/cache_headers_utils.h"
+
+#include <bits/stdint-intn.h>
```

Comment:
Why are these 'bits' headers needed? Are they portable?

Reply:
I did not add these, probably vscode extension, not the first time it did that but I did not notice these.
Will revise all the includes.